### PR TITLE
Prettier

### LINKED ISSUE #102: Prettier integration
I think the source of this addon is formatted with Prettier pretty consistently now - we should probably add a "styled with Prettier" badge to the README and enforce it in CI.

@anehx

### PR DESCRIPTION
**Pull request by anehx**
* Format all files with prettier
* Add precommit prettier
* Add app version to dummy app
* Add auto deploy with travis
* Add prettier as a dependency
* Enforce prettier with eslint

Fixes #102

### CODE REVIEW

File: tests/dummy/app/templates/components/color-label.hbs [hunk 1]
```diff
@@ -1,4 +0,0 @@
-<label style="color: green" for={{inputId}} class={{config.css.label}}>
-  {{yield}}
-  Custom label component!
-</label>
```
Comment (czosel) [inline tests/dummy/app/templates/components/color-label.hbs]: Should this really be deleted?

Comment (anehx) [reply tests/dummy/app/templates/components/color-label.hbs]: IMHO, yes. A custom label component is not something you'd want to show off in a demo app. It's enough to explain it in the readme.

Comment (anehx) [reply tests/dummy/app/templates/components/color-label.hbs]: Also it looked weird.

Comment (czosel) [reply tests/dummy/app/templates/components/color-label.hbs]: :smile: ok

**Pull request merged**